app/toolbar/components/status/states/Running.jsx

```diff
@@ -57,7 +60,6 @@ class Running extends React.Component<Props> {
         category: 'closeDocker'
```

Comment:
i think we should use the convention for stings to be ${action}.${object} eg 'close.docker'
```suggestion
        category: 'close.docker'
```